scripts/snapshot_check/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+# Docker compose file to run continuous Forest sync on both mainnet and calibnet.
+
+version: "3.7"
+
+services:
+  forest_calibnet:
+    image: ghcr.io/chainsafe/forest:${FOREST_TAG}
+    hostname: forest-calibnet
+    container_name: forest-calibnet
+    volumes:
+      - type: bind
+        source: ${BASE_FOLDER}
+        target: ${BASE_FOLDER}
+    entrypoint: ["/bin/bash", "-c"]
+    environment:
+      - SLACK_HOOK
+    command:
+      - |
+        apt-get install -y curl
+
+        report() {
+          curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"❌  $FOREST_HOSTNAME snapshot export failure!\"}" "$$SLACK_HOOK"
+        }
+        trap 'report' ERR
```

Comment:
to my understanding, this will indeed call `report()` in case of an error but then it will happily continue execution anyway, no? Shouldn't it be rather `report_and_die` with an exit?

Reply:
I thought it would die after reporting. My bash-fu is weak so it can't hurt to make sure.

Reply:
```bash
#!/bin/bash

report() {
  echo 'fiasco!'
}
trap 'report' ERR

if [ $# -eq 0 ]; then
  echo "begin"
  rmdir see-no-evil
  echo "carry on my wayward soooon"
fi
```
```
❯ bash tst.sh
begin
rmdir: failed to remove 'see-no-evil': No such file or directory
fiasco!
carry on my wayward soooon
```

Reply:
Ah, right. I'm missing the `set -e`. With that at the top, your example exits after the first error.